### DISCUSSION TITLE
help: CP data cann't sync to DP

### DISCUSSION THREAD

**Discussion by cczchunge** [apache/apisix#10400] [Q&A] [answered]
Apisix 3.2.1  in CP-DP Mode

new created routes will not be synced to DP:

in CP:
curl http://127.0.0.1:9180/apisix/admin/routes/x32 -H 'X-API-KEY: apisix' -X PUT -d '
{
    "methods": ["GET"],
    "uri": "/hmac-auth32",
    "host": "www.ccztest.com",
    "name": "ccztest32",
    "upstream": {
        "type": "roundrobin",
        "nodes": {
            "10.247.90.135:9000": 1
        }
    }
}'

in DP:
curl -v4 localhost:9080/hmac-auth32 -H "Host: www.ccztest.com"
* About to connect() to localhost port 9080 (#0)
*   Trying 127.0.0.1...
* Connected to localhost (127.0.0.1) port 9080 (#0)
> GET /hmac-auth32 HTTP/1.1
> User-Agent: curl/7.29.0
> Accept: */*
> Host: www.ccztest.com
> 
< HTTP/1.1 404 Not Found
< Date: Thu, 26 Oct 2023 02:47:20 GMT
< Content-Type: text/plain; charset=utf-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Server: APISIX/3.2.1
< 
{"error_msg":"404 Route Not Found"}

Environment:
- APISIX version (run `apisix version`):  3.2.1
- Operating system (run `uname -a`):  Linux iZ1pg01g9j9z3qotzmx9xeZ 3.10.0-1160.25.1.el7.x86_64 #1 SMP Wed Apr 28 21:49:45 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):   nginx version: openresty/1.21.4.1
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):  3.5.7
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`): /usr/local/bin/luarocks 3.9.2

**Comment by kayx23** [comment]
Can you afford to upgrade to 3.6.0? There's a breaking change in that release that refactored for standalone mode and subsequently resolved a few critical issues.

**Reply by cczchunge** [reply, marked as answer]
Thank you!
Problem is solved. 

#9263
